propolis/src/block.rs

```diff
@@ -202,7 +321,18 @@ impl<R: BlockReq> BlockDev<R> for PlainBdev<R> {
         BlockInquiry {
             total_size: self.sectors as u64,
             block_size: self.block_size as u32,
-            writable: !self.is_ro,
+            writable: !self.backing_store.is_ro(),
         }
     }
 }
+
+pub fn create_file_backed_block_device<R: BlockReq>(
```

Comment:
Minor: Since this is sort of a constructor for a `PlainBdev` I wonder if it should just be a static method on that type instead of a free-fn. We already have `PlainBdev::create` which just takes the backing store but we could make this another method like `from_file`. Thoughts?

Reply:
Nice, done in 6d1389c